### DISCUSSION TITLE
checkpoints file missing

### DISCUSSION THREAD

**Discussion by bg4ut4m** [anothermartz/Easy-Wav2Lip#42] [Q&A]
Getting the below error, where Can I find "predictor.pkl", pls help.

imports loaded!     
Traceback (most recent call last):
  File "/home/sagemaker-user/Easy-Wav2Lip/inference.py", line 117, in <module>
    with open(os.path.join('checkpoints','predictor.pkl'), 'rb') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'checkpoints/predictor.pkl'
Processing failed! :( see line above 👆
Processing failed

**Comment by bg4ut4m** [comment]
@anothermartz thanks for you help

**Comment by anothermartz** [comment]
These files are generated during the running of install.py

If you're running windows, please try the new [Easy-Wav2Lip.bat](https://github.com/anothermartz/Easy-Wav2Lip/releases/download/Easy-Wav2Lip-venvtest/Easy-Wav2Lip.bat) which should be able to check for all prerequestis and install them automatically if any are missing.

If on linux make sure you follow the instructions listed [here](https://github.com/anothermartz/Easy-Wav2Lip?tab=readme-ov-file#manual-installation).

Let me know how you get on :)

**Reply by bg4ut4m** [reply]
I followed all the steps there, however I am see that no file named "predictor.pkl". I cross checked the checkpoints folder as well, but for some reason that file is not present.

Opening config.ini - customize accordingly then save and close it to continue!
config.ini closed - starting Easy-Wav2Lip...
Processing _video.mp4 using audio.mp3 for audio
imports loaded!     
Traceback (most recent call last):
  File "/home/sagemaker-user/Easy-Wav2Lip/inference.py", line 117, in <module>
    with open(os.path.join('checkpoints','predictor.pkl'), 'rb') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'checkpoints/predictor.pkl'

**Reply by anothermartz** [reply]
There must be an issue when running install.py as that is where they're generated, can you run that and look for any errors in the terminal window?
